src/plugins/action_grant_permissions/src/grantPermissions.ts

```diff
@@ -0,0 +1,27 @@
+module.exports['grantPermissions'] = async function (
+  { username, permissionNames }: any,
+  DBConnect: any
+) {
+  try {
+    console.log('\nGranting permission/s:')
+    const currentUserPermisionsNames = (await DBConnect.getUserPermissionNames(username)).map(
+      ({ name }: any) => name
+    )
+
+    for (const permissionName of permissionNames) {
+      if (currentUserPermisionsNames.includes(permissionName)) continue
+      await DBConnect.joinPermissionNameToUser(username, permissionName)
+      currentUserPermisionsNames.push(permissionName)
+    }
+    return {
+      status: 'Success',
+      error_log: '',
```

Comment:
```suggestion
      error_log: '',
      output: {
        permissionNames,
     },
```
I think a minimal output object is useful and consistent with the other plugins. It's also useful for debugging purposes.